test/constructed_images.jl

```diff
@@ -354,4 +354,9 @@ mutable struct TestType end
         exportimagepixels!(buf2view, wand, cs, channelorder)
         @test buf2view == Ar
     end
+
+    @testset "issue #206" begin
+        img = ImageMagick.load("images/cameraman.tif")
```

Comment:
Can't you use `TestImages.jl` instead of adding a binary blob to this repository?

Reply:
`TestImages` depends on `ImageMagick`, so that would create a circular dependency

Reply:
What circular dependency?  As a test-only dependency?

Reply:
Ah, you're right, that should still work